india_forecast_app/app.py

```diff
@@ -268,7 +267,14 @@ def save_forecast(
     help="Set the python logging log level",
     show_default=True,
 )
-def app(timestamp: dt.datetime | None, write_to_db: bool, log_level: str):
+def app_click(timestamp: dt.datetime | None, write_to_db: bool, log_level: str):
```

Comment:
@peterdudfield with this change should this line https://github.com/openclimatefix/india-forecast-app/blob/main/Dockerfile#L44 in the dockerfile which specifies the entry point be modified to app_click instead of app?

Reply:
I think thats ok, it references to the file name

Reply:
Ok thanks, ah yep change it on the last line on `app.py` so that's fine